modules/perception/obstacle/camera/visualizer/frame_content.h

```diff
@@ -174,33 +166,31 @@ class FrameContent {
     return static_cast<int64_t>(d / kEpsilon_);
   }
 
-  double MapKeyToDouble(const int64_t key) {
-    return key * kEpsilon_;
-  }
+  double MapKeyToDouble(const int64_t key) { return key * kEpsilon_; }
 
   // input
   // 1.radar
-  std::map<int64_t, RadarContent> radar_caches_;
+  std::unordered_map<int64_t, RadarContent> radar_caches_;
   double current_radar_timestamp_;
 
   // 2.camera
-  std::map<int64_t, CameraContent> camera_caches_;
+  std::unordered_map<int64_t, CameraContent> camera_caches_;
   double current_camera_timestamp_;
 
   // 3.fusion
-  std::map<int64_t, FusionContent> fusion_caches_;
+  std::unordered_map<int64_t, FusionContent> fusion_caches_;
   double current_fusion_timestamp_;
 
   // 4.ground truth
-  std::map<int64_t, GroundTruthContent> gt_caches_;
+  std::unordered_map<int64_t, GroundTruthContent> gt_caches_;
   double current_gt_timestamp_;
 
   // 5.image
-  std::map<int64_t, ImageContent> image_caches_;
+  std::unordered_map<int64_t, ImageContent> image_caches_;
```

Comment:
caches is more convenient with map... I may use sorted property

Reply:
Also, the camera visualization tool is under reconstruction.. thanks

Reply:
I changed them back to map as you will need the sorted property.

Reply:
thanks!